test/helpers/view_components/mentoring/inbox_test.rb

```diff
@@ -4,23 +4,18 @@ class MentoringInboxTest < ViewComponentTestCase
   test "mentoring inbox rendered correctly" do
     conversations_request = { endpoint: "conversations-endpoint" }
     tracks_request = { endpoint: "tracks-endpoint" }
-    sort_options = [
-      { value: 'recent', label: 'Sort by Most Recent' },
-      { value: 'exercise', label: 'Sort by Exercise' },
-      { value: 'student', label: 'Sort by Student' }
-    ]
 
-    assert_component_equal ViewComponents::Mentoring::Inbox.new(
-      conversations_request, tracks_request, sort_options
-    ).to_s,
-                           { id: "mentoring-inbox", props: {
-                             conversations_request: { endpoint: "conversations-endpoint" },
-                             tracks_request: { endpoint: "tracks-endpoint" },
-                             sort_options: [
-                               { value: 'recent', label: 'Sort by Most Recent' },
-                               { value: 'exercise', label: 'Sort by Exercise' },
-                               { value: 'student', label: 'Sort by Student' }
-                             ]
-                           } }
+    component = ViewComponents::Mentoring::Inbox.new(
+      conversations_request, tracks_request
+    ).to_s
```

Comment:
I pulled this out of the assert bit below as I found it really hard to read having that code and the hash there as well.